OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -985,9 +1105,42 @@ function Add-TentacleToWorkerPool
 
         [Parameter()]
         [String[]]
-        $workerPools
+        $workerPools,
+
+        [Parameter()]
+        [String]
+        $Space,
+
+        [Parameter()]
+        [String]
+        [ValidateSet("Listen", "Poll")]
+        $CommunicationsMode = "Listen"
     )
 
+    # Declare local variables
+    $commsStyle = ""
+
+    # Translate communications mode
```

Comment:
These comments... they just need to die. It's taken 22 lines to write something that could take 5:
```
    if ($communicationsMode -eq "Listen") {
        $commsStyle = "TentaclePassive"
    } else {
        $commsStyle = "TentacleActive"
    }
```
We're aiming for concise, yet clear, understandable code. If it needs comments, then the code should be re-written until it doesn't need comments.